simple-dns/src/dns/rdata/txt.rs

```diff
@@ -66,26 +67,23 @@ impl<'a> TXT<'a> {
     pub fn attributes(&self) -> HashMap<String, Option<String>> {
         let mut attributes = HashMap::new();
 
-        for char_str in &self.strings {
-            let mut splited = char_str.data.splitn(2, |c| *c == b'=');
-            let key = match splited.next() {
-                Some(key) => match std::str::from_utf8(key) {
-                    Ok(key) => key.to_owned(),
-                    Err(_) => continue,
-                },
-                None => continue,
-            };
+        let full_string: String = (*self).clone().into();
```

Comment:
I think it is better to create a new function, maybe `long_attributes`, rather than changing the existing functionality, because there may be people relying on the current behavior